### PR TITLE
Add Velocity Support

### PR DESCRIPTION
**Pull request by Alijkaz**
Implemented and tested Velocity support for ChatSentinel (currently using it in a network)
Tested configurations, commands and modules, All seems to work fine

**Note:** I've used a library called [Simple-Yaml](https://github.com/Carleslc/Simple-YAML) for velocity support to make things easier and more readable (instead of directly using SnakeYaml) but in other hand, It adds up around ~400kb to the size of the plugin

### CODE REVIEW

**Comment by Alijkaz** [comment]
Making #10 happen :)

**Comment by linsaftw** [comment]
Wouldn't it be easier to use the already present ConfigUtil instead of using another YAML library?

File: src/main/java/dev/_2lstudios/chatsentinel/velocity/listeners/ChatListener.java [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+	public ChatListener(final ProxyServer proxyServer, final ModuleManager moduleManager,
+                        final ChatPlayerManager chatPlayerManager) {
+		this.proxyServer = proxyServer;
+		this.moduleManager = moduleManager;
+		this.chatPlayerManager = chatPlayerManager;
+	}
+
+	private void sendMessage(final CommandSource source, final String message) {
+		source.sendMessage(Component.text(message.replace('&', '§')));
```
Comment (Marlonzockt) [inline src/main/java/dev/_2lstudios/chatsentinel/velocity/listeners/ChatListener.java:35]: Here you can use adventure's LegacyComponentSerializer:
```java
LegacyComponentSerializer.legacyAmpersand()
```

Comment (Alijkaz) [reply src/main/java/dev/_2lstudios/chatsentinel/velocity/listeners/ChatListener.java]: Updated in the last push, Thanks!

**Comment by Alijkaz** [comment]
Hey, sorry for the delay, I did not have enough time to update the pr
I am now using the velocity yaml libraries instead of the external library. The plugin has been tested and it seems to work fine

**Comment by linsaftw** [comment]
Thank you very much. I will test it later and if it works fine i will merge

**Comment by linsaftw** [comment]
Seems we have to update the velocity support, Right? I am interested on this :)

**Comment by Thin-Troll** [comment]
Where can I find the latest build of ChatSentinel on Velocity?
I downloaded the last build from discord.. I don't know if you updated it on github.
image


I would like to see this on Velocity, because this proxy gives MUCH more performance, it's just even smoother on the eye.
Banal movement in the lobby - gives happiness.

**Comment by linsaftw** [comment]
> Where can I find the latest build of ChatSentinel on Velocity? I downloaded the last build from discord.. I don't know if you updated it on github. image
> 
> I would like to see this on Velocity, because this proxy gives MUCH more performance, it's just even smoother on the eye. Banal movement in the lobby - gives happiness.

This pull needs to be updated. it doesnt work there yet.

**Pull request closed without merging**